File: python/bee1010.py
[cod1,qnt1,price1] = input().split(' ')
[cod2,qnt2,price2] = input().split(' ')
total = (int(qnt1)*float(price1)) + (int(qnt2)*float(price2))
print(f'VALOR A PAGAR: R$ {total:.2f}')

File: js/bee1010.js
var input = require('fs').readFileSync('js/stdin', 'utf8');
var lines = input.split('\n');
var [cod1,und1,vlr1] = lines[0].split(' ').map(Number)
var [cod2,und2,vlr2] = lines[1].split(' ').map(Number)
var total = (und1*vlr1) + (und2*vlr2)

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`)